storage/src/vespa/storage/persistence/persistencethread.cpp

```diff
@@ -186,9 +185,9 @@ PersistenceThread::handleGet(api::GetCommand& cmd)
     document::FieldSetRepo repo;
     document::FieldSet::UP fieldSet = repo.parse(*_env._component.getTypeRepo(), cmd.getFieldSet());
     // _context is reset per command, so it's safe to modify it like this.
-    _context.setReadConsistency(api_read_consistency_to_spi(cmd.internal_read_consistency()));
+    context.setReadConsistency(api_read_consistency_to_spi(cmd.internal_read_consistency()));
```

Comment:
Consider removing now outdated comment for this line regarding old `_context`

Reply:
Fixed